oneflow/core/framework/layout.cpp

```diff
@@ -0,0 +1,48 @@
+/*
+Copyright 2020 The OneFlow Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+#include "oneflow/core/common/util.h"
+#include "oneflow/core/common/container_util.h"
+#include "oneflow/core/framework/layout.h"
+#include "oneflow/core/common/preprocessor.h"
+
+namespace oneflow {
+
+Symbol<Layout> Layout::Get(LayoutType layout_type) {
+  static HashMap<LayoutType, Symbol<Layout>> layouttype2layout{
+#define MAKE_ENTRY(layout_type) {OF_PP_CAT(LayoutType::k, layout_type), layout_type()},
+      OF_PP_FOR_EACH_TUPLE(MAKE_ENTRY, LAYOUT_SEQ)
+#undef MAKE_ENTRY
+  };
+  return CHECK_JUST(MapAt(layouttype2layout, layout_type));
```

Comment:
```suggestion
  return layout_type2layout[layout_type];
```
因为已经保证了合法性
其他的三个 map 同理